luajit/VITABUILD

```diff
@@ -0,0 +1,29 @@
+pkgname=luajit
+pkgver=2.0.5
+pkgrel=1
+url="https://github.com/hyln9/vita-luajit"
+source=(
+  "git+https://github.com/hyln9/vita-luajit.git#commit=0f0252ab8615bc4b097da0bfe0bf10d0666a3dcd"
+)
+sha256sums=(
+  'SKIP'
+)
+depends=('vita-libdl')
+
+build() {
+  cd vita-luajit/src
+  make CFLAGS="-DLUAJIT_USE_SYSMALLOC -DLJ_FFI_STUB -DLUAJIT_DISABLE_JIT" HOST_CC="gcc -m32" CROSS=arm-vita-eabi- TARGET=arm BUILDMODE=static TARGET_SYS=PSP2 PREFIX="${VITASDK}/arm-vita-eabi" TARGET_FLAGS="-fno-optimize-sibling-calls"
```

Comment:
1) PREFIX on build stage, according to readme, should point to target system prefix (it's one of the paths where lua will search for modules.
2) Don't use `${VITASDK}/arm-vita-eabi`, use `$prefix`

Reply:
Oh, you're right the prefix is wrong according to the documentation. Good catch.